src/vector_topic_modeling/distributed.py

```
@@ -0,0 +1,191 @@
+            a_i = 0.0
+
+        # Calculate b_i
+        b_i = float("inf")
+        for c_j, other_cluster_indices in enumerate(cluster_indices):
+            if c_i == c_j or not other_cluster_indices:
+                continue
+            mean_dist = sum(
+                1.0 - cosine_similarity(v_i, vectors[j]) for j in other_cluster_indices
+            ) / len(other_cluster_indices)
+            if mean_dist < b_i:
+                b_i = mean_dist
```

Ruff (0.15.7)

[warning] Replace `if` statement with `b_i = min(b_i, mean_dist)`

Replace with `b_i = min(b_i, mean_dist)`

(PLR1730)